source/process/community-guidelines.rst

```diff
@@ -100,3 +130,76 @@ Response Writing Tips
 - **Be thankful**
   - Communities really respond well to being praised and thanked for their work
+  
+Sample Responses
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+General Issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. If an issue has a PR submitted by community, but no associated Jira ticket, leave it open until the PR is merged
+
+2. If an issue has a Jira ticket with a ``help-wanted`` label, there is a Help Wanted ticket in GitHub. It can be closed with the following note:
+
+.. code-block:: text
+  Hi @username
+
+  Thanks for the report! We have created a [Help Wanted issue here](link to GitHub issue) and are looking for community's help. Would you be interested helping with a pull request?
+
+3. If an issue has a Jira ticket without a ``help-wanted`` label and assigned to the current release fix version for a developer to fix, it can be closed with the following note
+
+.. code-block:: text
+
+  Hi @username
+
+  Thanks for the report! We have created a [Jira ticket](link to Jira ticket) to track it. If you're interested helping with a pull request, please let us know.
+
+4. If an issue has a Jira ticket without a ``help-wanted`` label but not assigned to the current release fix version, queue Jira ticket back to triage to ask if a help wanted issue could be created for it.
+
+5. If the reporter doesn’t respond in two weeks, close the issue with the following note:
+
+.. code-block:: text
+
+  Hi @username, we haven't received an update so we'll assume that the problem is fixed or is no longer valid. 
+
+  If you still experience the same problem, try upgrading to the latest version. 
+
+  If the issue persists, reopen this issue with the relevant information and we'd be glad to help you where we can.
+
+Feature Requests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Respond to the issue with the following note
+
+.. code-block:: text
+
+  Thanks, appreciate your feedback @{username}.  
+
+  Would you like to [contribute this in the feature idea forum](https://mattermost.uservoice.com/forums/306457-general/) so it can be discussed, upvoted and considered for a [help wanted ticket](https://docs.mattermost.com/process/help-wanted.html)?
+
+  Please include a link back to this GitHub issue. If you're interested in implementing, please say so and we'll prioritize the review. 
+
+  You get **10** votes in the feature idea forum, and each one influences the future of the project.
+
+Hard Questions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To respond to tough questions, use the `SCIPAB method <https://www.mandel.com/top-ten-reasons/tools-methodology/scipab>`_ to help formulate a response and send for community lead to review. 
+
+Situation:
+  State what you know about your listeners' circumstances that is relevant to your discussion or presentation, e.g., current state of their business, technology, industry, or plans. 
```

Comment:
"...circumstances that is relevant..." > "...circumstances that are relevant..."